docs/recipes/auth/example-lh-auth.js

```diff
@@ -0,0 +1,43 @@
+const puppeteer = require('puppeteer');
+const lighthouse = require('lighthouse');
+
+const PORT = 8041;
```

Comment:
the readme exists but a `@fileoverview` would probably be helpful for each of these files (even if only a sentence and/or link back to the readme) to quickly give context to what a reader is looking at (and if it's what they're looking for)